src/main/java/com/ericsson/ei/mongodbhandler/MongoDBHandler.java

```diff
@@ -277,15 +284,17 @@ public void createTTLIndex(String dataBaseName, String collectionName, String fi
         if (mongoClient == null)
             return null;
         MongoDatabase db = mongoClient.getDatabase(dataBaseName);
-        List<String> collectionList = db.listCollectionNames().into(new ArrayList<String>());
+        List<String> collectionList = db.listCollectionNames()
+                                        .into(new ArrayList<String>());
         if (!collectionList.contains(collectionName)) {
             log.debug("The requested database(" + dataBaseName + ") / collection(" + collectionName
                     + ") not available in mongodb, Creating ........");
             try {
                 db.createCollection(collectionName);
             } catch (MongoCommandException e) {
                 String message = "collection '" + dataBaseName + "." + collectionName + "' already exists";
-                if (e.getMessage().contains(message)) {
+                if (e.getMessage()
+                     .contains(message)) {
```

Comment:
Why new line here? It is a short line ,  so okay to have if statement int one line.

Reply:
The new eiffel autoformatter does that.

Reply:
ok, was this formatter rule in the old formatter as well?
 I think we keep it on one line when is short like this, normally.

Reply:
Done